lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -1,12 +1,35 @@
 // @TODO: Move to Forge if this is shipped.
 .counter {
-  display: inline-block;
   color: $med-gray;
   font-weight: $weight-bold;
-  font-size: 13px !important;
-  padding-left: 3px;
+  font-size: 16px;
+  padding-left: 4px;
 
   &.is-active {
-    color: $purple;
+    color: $dark-gray;
+  }
+}
+
+.photo__actions li {
+  * {
+    display: inline;
+    vertical-align: middle;
+  }
+
+  img {
+    width: 30px !important;
+    height: 26px;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    &.is-active {
+      content: url("../images/pink-heart.svg");
+    }
+
+    &:not(.is-active) {
```

Comment:
I don't believe IE8 or IE9 support the `:not` operator, and (unfortunately!) we still support those. Would it be possible to have this as the default styling for the element and just override it if the `.is-active` class is applied?